### DISCUSSION TITLE
Failed to push a fix

### DISCUSSION THREAD

**Discussion by liorj-orca** [Checkmarx/kics#5532] [General] [answered]
Hi all, 
I followed the instructions at https://github.com/Checkmarx/kics/blob/master/docs/CONTRIBUTING.md and tried to push my change as mentioned using:
`git push --set-upstream origin <name-of-your-issue>`
However, I'm getting:
> ERROR: Permission to Checkmarx/kics.git denied to liorj-orca.
> fatal: Could not read from remote repository.
> 
> Please make sure you have the correct access rights
> and the repository exists.
anything special need to be done?

**Comment by kaplanlior** [comment, marked as answer]
You should push to **your fork** of KICS (e.g. https://github.com/liorj-orca/kics ) and submit a PR from there.
